styles.css

```diff
@@ -85,11 +165,27 @@ h2 {
 
 .past-activity {
   height: 100%;
-  width: 40vw;
+  width: 35vw;
   padding: 0px 50px 100px 50px;
 }
 
-.past-activity-card {
+.past-activity-log {
   height: 100%;
   width: 100%;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  /* margin-top: 40px; */
+}
+
+/*** OTHER ***/
+
+button:hover {
+  cursor: pointer;
```

Comment:
I think this would be really great to see a little scale added to the hover function so the button(s) become 5-10% larger when hovered over to get the user to recognize they're able click on it.  Might help a bit more with design semantics.  We can discuss as a team during our stand-up.